chains/sophonsepolia/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-explorer-verify.testnet.sophon.xyz/contract_verification
+    family: etherscan
+    name: Sophon Testnet Explorer
+    url: https://explorer.testnet.sophon.xyz
+chainId: 531050104
```

Comment:
add deployer info
```suggestion
chainId: 531050104
deployer:
  name: NAME
  url: LINK
```

Reply:
Is this what we should put there?

deployer:
  name: Abacus Works
  url: https://www.hyperlane.xyz

Reply:
with tx fusion deployer details